hw_9.py

```diff
@@ -0,0 +1,76 @@
+# 1. Використати файл як базу даних для збереження записів телефонної книги із попередніх завдань.
+import json
+phone_book = {'Vadym':'0665699589'}
+while True:
```

Comment:
You can open file only in cases when you need to update the data. 
For other purposes, you can use a dictionary as before, but load it by reading from your file at the beginning.
But don't forget to make sure the file exists and init your phone_book as an empty dict. Something like this:
```
try:
    with open('file', 'r') as file:
        data = file.read()
        phone_book = json.dumps(data)
except FileNotExistError:
    phone_book = {}

Reply:
And then, you can remove some using of file and use your `phone_book` for list, stats and show commands

Reply:
Your example help me to short my code. Thanks. I just add in except method opening file in "w+" mode. So if file doesn't exist it create a new one.
@klymenok check please